examples/computeToFandTracers.cpp

```diff
@@ -120,18 +120,42 @@ namespace {
     }
 
     Opm::FlowDiagnostics::Toolbox
-    initialiseFlowDiagnostics(/* mutable */ Opm::ECLGraph& G, const int step)
+    initialiseFlowDiagnostics(/* mutable */ Opm::ECLGraph& G,
```

Comment:
Since you're in this area, you can change the `/* mutable */` to a real `const` here.